rancher/install-rancher.sh

```diff
@@ -292,7 +292,8 @@ helm install rancher rancher-stable/rancher \
    --namespace cattle-system \
    --set hostname=$HOST \
    --set ingress.tls.source=letsEncrypt \
-   --set letsEncrypt.email=$EMAIL
+   --set letsEncrypt.email=$EMAIL \
+   --set bootstrapPassword="bytlosen123"
```

Comment:
```suggestion
   --set bootstrapPassword=$BOOTSTRAPPASSWORD
```

Along with changing 

```diff
- if [ "$#" -ne 5 ]
+ if [ "$#" -ne 6 ]
```

On line 5 and adding a new argument below line 16?

```diff
  USERNAME=$5
+ BOOTSTRAPPASSWORD=$6
```

Reply:
Along with a change to the yaml-file?

```diff
-   - ./install-rancher.sh {{params.hostname}} {{params.email}} {{params.project}} {{params.apikey}} {{users.0.username}}
+   - ./install-rancher.sh {{params.hostname}} {{params.email}} {{params.project}} {{params.apikey}} {{users.0.username}} {{bootstrappassword}}

Reply:
I have applied the changes to the PR in a new commit